src/main/java/org/example/jakarta_labb/resource/MovieResource.java

```diff
@@ -0,0 +1,57 @@
+package org.example.jakarta_labb.resource;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import org.example.jakarta_labb.entity.Movie;
+import org.example.jakarta_labb.repository.MovieRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Path("/movies")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class MovieResource {
+
+    private final MovieRepository movieRepository;
+
+    @Inject
+    public MovieResource(MovieRepository movieRepository) {
+        this.movieRepository = movieRepository;
+    }
+
+    @GET
+    public List<Movie> getAllMovies() {
```

Comment:
I don't think we should expose the entity.
Please use the movie DTO in all endpoints 😃

Reply:
was thinking i could take care of that in my next PR when we have a MovieDto class to refer to.

Reply:
Sounds great!